moped-editor/src/utils/signalComponentHelpers.js

```diff
@@ -19,21 +19,17 @@ export const getSignalOptionSelected = (option, value) =>
  * MUI autocomplete getOptionLabel function to which formats the value rendered in
  * the select option menu
  */
-export const getSignalOptionLabel = option =>
+export const getSignalOptionLabel = (option) =>
   // this label formatting mirrors the Data Tracker formatting
   `${option.properties.signal_id}: ${option.properties.location_name}`;
 
 /**
  * Imitate a "drawn point" feature from a traffic signal geojson feature. Sets required
- * fields so that featureCollection can be used in the DB mutation on submit
+ * fields so that a Knack feature can be inserted into the feature_signals table
```

Comment:
Lots of these changes are making the signals records that we fetch from the Socrata dataset (and originate from Data Tracker) into the format expected by the new `feature_signals` table instead storing the signal as a GeoJSON feature collection like we currently do. The existing helpers were super handy and were straightforward to update.